x/gamm/types/constants.go

```diff
@@ -9,6 +9,8 @@ const (
 	MaxPoolAssets = 8
 
 	OneShareExponent = 18
+
+	SigFigsExponent = 8
```

Comment:
Can we add a comment explaining what this variable is and what it's used for?

Reply:
Done!